packages/react-vapor/package.json

```diff
@@ -55,7 +53,7 @@
         "react-tether": "1.0.5",
         "react-textarea-autosize": "8.3.3",
         "react-transition-group": "2.9.0",
-        "reselect": "4.1.4",
+        "reselect": "4.1.2",
```

Comment:
There was [an error](https://github.com/reduxjs/reselect/issues/554) with version 4.1.4, but it has been fixed in 4.1.5, I will update to the latest.